tools/ffi_coverage/src/allowlist.rs

```diff
@@ -274,7 +274,7 @@ lazy_static::lazy_static! {
         "icu::list::FormattedList",
 
         // Experimental
-        "icu::casemapping",
+        "icu::casemap",
```

Comment:
this will conflict with my PR; and is also going to flood missing_apis.txt and fail your tests

Reply:
it shouldn't, because the `icu` module is also renamed

Reply:
oh, wait, I see